app/serializers/user_serializer.rb

```diff
@@ -1,3 +1,3 @@
 class UserSerializer < ActiveModel::Serializer
-  attributes :id, :name, :image, :email
+  attributes :id, :name, :image, :email, :oauth_token, :oauth_expires_at
```

Comment:
I don't know that this is the way to do this, but we can remove it later after testing.